geocoder_tester/world/france/test_synonymes.csv

```diff
@@ -0,0 +1,7 @@
+query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_city
+2 rue du 10e niort,,,4,,2,Rue du Dixième,Niort
```

Comment:
this one is tricky, bragi match `Paris 10e Arrondissement` :laughing: 

only addock manage it.
nice, but difficult test!